openfl-tutorials/interactive_api/HPU/PyTorch_Kvasir_UNet/README.md

```diff
@@ -4,21 +4,102 @@
 #### The name of the file/example that contain HPU adaptations start with "HPU". 
 For example: PyTorch_Kvasir_UNet.ipynb placed under workspace folder contains the required HPU adaptations.
 
- All the execution steps mention in last section (**V. How to run this tutorial**) remain same for HPU examples but as pre-requisite it needs some additional environment setup and Habana supported package installations which is explained below from **section I to IV**.
+ All the execution steps mention in last section (**V. How to run this tutorial**) remain same for HPU examples but as pre-requisite it needs some additional environment setup and Habana supported package installations which is explained below from **section I to V**.
 
  **Note:** By default these experiments utilize 1 HPU device
 
  <br/>
 
- ## **I. AWS DL1 Instance Setup**
+ ## **I. Intel Developer Cloud Setup**
+This example was test on the Intel Developer Cloud utilizing Gaudi2 instance. 
+
+For accessing the Gaudi2 instances on the Intel Developer Cloud follow the instructions [here](https://developer.habana.ai/intel-developer-cloud/)
+
+The Gaudi instance in the Intel Developer Cloud comes SynapseAI SW Stack for Gaudi2 installed. Skip sections (**II. , III.***)
+
+Further more our testing was done using the habana based Docker container built using the Dockerfile base discussed below:
+
+Let's create a Dockerfile with the following content and name it Dockerfile_Habana:
+
+```
+
+FROM vault.habana.ai/gaudi-docker/1.10.0/ubuntu20.04/habanalabs/pytorch-installer-2.0.1/latest
+
+ENV HABANA_VISIBLE_DEVICES=all
+ENV OMPI_MCA_btl_vader_single_copy_mechanism=none
+
+ENV DEBIAN_FRONTEND="noninteractive"  TZ=Etc/UTC
+RUN apt-get update && apt-get install -y tzdata bash-completion \
+        #RUN apt update && apt-get install -y tzdata bash-completion \
+        python3-pip openssh-server vim git iputils-ping net-tools curl bc gawk \
+        && rm -rf /var/lib/apt/lists/*
+
+
+RUN pip install numpy \
+        && pip install jupyterlab \
+        && pip install matplotlib \
+        && pip install openfl
+
+
+RUN git clone https://github.com/securefederatedai/openfl.git /root/openfl
+
+WORKDIR /root
+ 
+```
+
+This base container comes with HPU Pytorch packages already installed.  Hence you could skip step: **IV.** below. 
+
+Build the above container and then launch it using:
+
+```
+export Container_Image="myrepo_name:gaudi-docker-ubuntu20.04-openfl"
+
+docker build -t ${container_image}  -f Dockerfile_Habana .
+docker run  --net host -id --name openfl_gaudi_run  ${Container_Image} bash
```

Comment:
Capitalization should be consistent to make this work with copy-paste and would help user understanding to make env variable name more meaningful
```suggestion
export GAUDI_DOCKER_IMAGE="gaudi-docker-ubuntu20.04-openfl"

docker build -t ${GAUDI_DOCKER_IMAGE} -f Dockerfile_Habana .
docker run  --net host -id --name openfl_gaudi_run  ${GAUDI_DOCKER_IMAGE} bash
```